### PR TITLE
feat(agw): fix gtp tunnels for containerized agw

### PR DESCRIPTION
**Pull request by ymasmoudi**
Signed-off-by: YOUSSEF EL MASMOUDI <ymasmoudi@fb.com>

## Summary

reset gtp tunnels upon restart of sctpd
disable grpc debug mode

## Test Plan

**** TOTAL number of test cases:1

```
Check if a test case can be started: if state == hybrid.TC_READYTORUN
Run test case (1) on hybrid: Hybrid_40_4G_IMSI_Attach_&_Traffic()
Wait for the test case to finish or for a timeout: hybrid.tcwait(hybrid.TC_READYTORUN,60)
result: True
Verdict(Hybrid_40_4G_IMSI_Attach_&_Traffic) = VERDICT_PASS

**** Finished testcase **************

Testlist verdict = VERDICT_PASS
Closing NG40 logger: Name = hybrid_40_211028_095629-1; ID = 1
 >>>>>>>> close <<<<<<<<<
```

```
[DEBUG] Success logging event: s1_setup_success
[DEBUG] Success logging event: s1_setup_success
[INFO] Sending S6A-AUTHENTICATION_INFORMATION_REQUEST with IMSI: 222456000000101
[INFO] Received S6A-AUTHENTICATION_INFORMATION_ANSWER for IMSI: 222456000000101; Status: ; StatusCode: 2001
[INFO] Sending S6A-AUTHENTICATION_INFORMATION_REQUEST with IMSI: 222456000000101
[INFO] Received S6A-AUTHENTICATION_INFORMATION_ANSWER for IMSI: 222456000000101; Status: ; StatusCode: 2001
[DEBUG] Sending S6A-UPDATE_LOCATION_REQUEST with IMSI: 222456000000101
[INFO] Received S6A-LOCATION-UPDATE_ANSWER for IMSI: 222456000000101; Status: ; StatusCode: 2001
[INFO] sent itti S6A-LOCATION-UPDATE_ANSWER for IMSI: 222456000000101
Received SGW IP Address 0x6290003b0028
PCEF Client thread id 139952292505344[DEBUG] Success logging event: attach_success
[DEBUG] Success logging event: detach_success
```

### CODE REVIEW

File: lte/gateway/deploy/roles/agw_docker/tasks/main.yml [hunk 1]
```diff
@@ -144,6 +144,14 @@
   tags:
     - agwc
 
+- name: Restart ovs
+  ignore_errors: yes
+  shell: |
+    ifdown gtp_br0
+    /etc/init.d/openvswitch-switch  force-reload-kmod
```
Comment (pshelar) [inline lte/gateway/deploy/roles/agw_docker/tasks/main.yml]: There is script to update OVS, lets use that `ovs-kmod-upgrade.sh`

File: lte/gateway/docker/docker-compose.yaml [hunk 2]
```diff
@@ -141,7 +141,10 @@ services:
       - seccomp:unconfined
     environment:
       MAGMA_PRINT_GRPC_PAYLOAD: 0
-    command: /usr/local/bin/sctpd
+    command: >
+      sh -c "/usr/local/bin/magma-bridge-reset.sh &&
+            /etc/init.d/openvswitch-switch load-kmod &&
+            /usr/local/bin/sctpd"
```
Comment (pshelar) [inline lte/gateway/docker/docker-compose.yaml:147]: magma-bridge-reset.sh takes care of loading kmod, there is no need to do it here.

**Review by pshelar — CHANGES_REQUESTED** [review]

File: lte/gateway/docker/docker-compose.yaml (hunk 2 above)
Comment (ymasmoudi) [reply lte/gateway/docker/docker-compose.yaml:147]: @pshelar actually, magma-bridge-reset.sh call force-reload-kmod. This is would not work from within the container.

File: lte/gateway/deploy/roles/agw_docker/tasks/main.yml (hunk 1 above)
Comment (ymasmoudi) [reply lte/gateway/deploy/roles/agw_docker/tasks/main.yml]: fixed

File: lte/gateway/docker/docker-compose.yaml (hunk 2 above)
Comment (pshelar) [reply lte/gateway/docker/docker-compose.yaml:147]: why `openvswitch-switch load-kmod` works, but `force-reload-kmod` does not?

**Pull request closed without merging**